packages/jellyfish-api-core/__tests__/category/icxorderbook/createOrder.test.ts

```diff
@@ -0,0 +1,174 @@
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import {
+  ICXGenericResult, ICXOfferInfo, ICXOrderInfo, ICXOrder, InputUTXO
+} from '../../../src/category/icxorderbook'
+import BigNumber from 'bignumber.js'
+import { accountDFI, checkBTCSellOrderDetails, checkDFISellOrderDetails, idDFI, setup } from './common.test'
+import { RpcApiError } from '../../../src'
+
+describe('Should test ICXOrderBook.createOrder', () => {
```

Comment:
```suggestion
describe('ICXOrderBook.createOrder', () => {
```

In JS ecosystem BDD style, describe just describe the behavior without having a verb.

```ts
describe('context of describe') <- no verb
  it ('should do something') <- contains verb
```

All lowercased unless you are specifying code component/interface.